Adapter.H

```diff
@@ -1,25 +1,48 @@
 #ifndef PRECICEADAPTER_H
 #define PRECICEADAPTER_H
 
-#include "Interface.H"
+#ifdef UNIT_TESTING
+    #include "Tests/UnitTests/AdapterTest/AdapterUnittestIncludes.h"
```

Comment:
@huangq1234553 what happened to the `Tests/UnitTests` directory? Are these exactly the same files as in #118?